basin3d/core/schema/query.py

```diff
@@ -118,13 +136,34 @@ def __init__(self, **data):
         :param data: the data
         """
 
-        # convert strings to lists for some fields
-        for field in ["monitoring_features", "observed_property_variables", "monitoringFeatures",
-                      "observedPropertyVariables"]:
+        # convert strings to lists for some fields; the camel case are for Pydantic validation (don't delete)
+        for field in ["monitoring_feature", "monitoringFeature", "observed_property", "observedProperty",
+                      "statistic", "result_quality", "sampling_medium"]:
             if field in data and data[field] and isinstance(data[field], str):
                 data[field] = list([data[field]])
+
+        data = self.__validate__(**data)
+
         super().__init__(**data)
 
+    @staticmethod
+    def __validate__(**data):
+        """
+        Valiate
+        :return:
+        """
+        if 'aggregation_duration' in data and data['aggregation_duration'] is None:
+            del data['aggregation_duration']
+        return data
+
+    def get_mapped_fields(self) -> list:
+        # observed_property_variables is first b/c it is most likely to have compound mappings.
+        # ToDo: check how order may affect translation (see core/synthesis)
+        return ['observed_property', 'aggregation_duration', 'statistic', 'result_quality', 'sampling_medium']
+
+    def get_prefixed_fields(self) -> list:
+        return ['monitoring_feature']
+
```

Comment:
This should be a ClassVar field (from pydantic)
```suggestion
    prefixed_fields: ClassVar[List[str]] = ['monitoring_feature']

```

Reply:
Done. Changed get_mapped_fields similarly.